### PR TITLE
Updated and new disputed records

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/1215.

This PR brings in new disputed records from Natural Earth, adds properties and alt-geometries for existing disputed records already in WOF, and corrects properties of existing records in WOF.

Additions include updated hierarchies, new name translations, new concordance values, new geometries, and standardized properties (like parent_id and country codes). 

Additional work is needed to take care of Jammu/Kashmir and some of the disputed areas in India, but that work will take place through #170 / #169 / #496.

### CODE REVIEW

File: data/115/933/952/9/1159339529.geojson [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+    "geom:bbox":"28.501081584,46.553128561,29.95262089,48.155950623",
+    "geom:latitude":47.322637,
+    "geom:longitude":29.312036,
+    "iso:country":"XX",
+    "lbl:latitude":47.290691,
+    "lbl:longitude":29.252183,
+    "mps:latitude":47.290691,
+    "mps:longitude":29.252183,
+    "mz:hierarchy_label":1,
+    "mz:hierarchy_label_country":0,
```
Comment (stepps00) [inline data/115/933/952/9/1159339529.geojson]: Similar to Donblass to set all these label flags to 1.

File: data/115/933/953/3/1159339533.geojson [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+    "geom:bbox":"-5.35838675846,36.110500393,-5.33877348324,36.141119672",
+    "geom:latitude":36.129464,
+    "geom:longitude":-5.3467,
+    "iso:country":"XX",
+    "lbl:latitude":36.129426,
+    "lbl:longitude":-5.3467,
+    "mps:latitude":36.129426,
+    "mps:longitude":-5.3467,
+    "mz:hierarchy_label":1,
+    "mz:hierarchy_label_country":0,
```
Comment (stepps00) [inline data/115/933/953/3/1159339533.geojson]: Let's let this country name through with all flags being 1.

File: data/115/933/953/3/1159339533.geojson [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+    "wof:supersedes":[],
+    "wof:tags":[]
+},
+  "bbox": [
+    -5.35838675846141,
+    36.11050039300005,
+    -5.33877348324262,
+    36.14111967199992
+],
+  "geometry": {"coordinates":[[[-5.35838675846141,36.14110944700037],[-5.33877348324262,36.14111967199992],[-5.33991451699995,36.12982819200005],[-5.33906002499992,36.12384674700007],[-5.34203040299991,36.11050039300005],[-5.35024980399993,36.11928945500006],[-5.35838675846141,36.14110944700037]]],"type":"Polygon"}
```
Comment (stepps00) [inline data/115/933/953/3/1159339533.geojson]: Store this as NE alt geom, but copy the geom from the country as default here, and set same src as there.

File: data/115/933/953/5/1159339535.geojson [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+    "geom:bbox":"62.800641723,45.565824483,63.96713383,46.390838725",
+    "geom:latitude":45.97833,
+    "geom:longitude":63.383897,
+    "iso:country":"XX",
+    "lbl:latitude":45.978332,
+    "lbl:longitude":63.383897,
+    "mps:latitude":45.978332,
+    "mps:longitude":63.383897,
+    "mz:hierarchy_label":1,
+    "mz:hierarchy_label_country":0,
```
Comment (stepps00) [inline data/115/933/953/5/1159339535.geojson]: Let's set this and other label flags to 1 as it's a lease area, people don't disagree about which country.

File: data/115/933/955/1/1159339551.geojson [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+    "geom:bbox":"72.357676629,-7.432224217,72.494639519,-7.226739191",
+    "geom:latitude":-7.331575,
+    "geom:longitude":72.437985,
+    "iso:country":"XX",
+    "lbl:latitude":-7.297663,
+    "lbl:longitude":72.398825,
+    "mps:latitude":-7.297663,
+    "mps:longitude":72.398825,
+    "mz:hierarchy_label":1,
+    "mz:hierarchy_label_country":0,
```
Comment (stepps00) [inline data/115/933/955/1/1159339551.geojson]: Lease area, set all label flags to 1.

File: data/115/933/952/5/1159339525.geojson [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+    "geom:bbox":"-75.232858853,19.891142236,-75.094950291,19.97158369",
+    "geom:latitude":19.929254,
+    "geom:longitude":-75.154673,
+    "iso:country":"XX",
+    "lbl:latitude":19.918398,
+    "lbl:longitude":-75.117277,
+    "mps:latitude":19.918398,
+    "mps:longitude":-75.117277,
+    "mz:hierarchy_label":1,
+    "mz:hierarchy_label_country":0,
```
Comment (stepps00) [inline data/115/933/952/5/1159339525.geojson]: Lease area so all label flags to 1.

File: data/115/933/950/3/1159339503.geojson [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+    "geom:bbox":"126.660516398,37.782182705,128.394783581,38.624335028",
+    "geom:latitude":38.245854,
+    "geom:longitude":127.517532,
+    "iso:country":"XX",
+    "lbl:latitude":38.274654,
+    "lbl:longitude":127.517738,
+    "mps:latitude":38.274654,
+    "mps:longitude":127.517738,
+    "mz:hierarchy_label":1,
+    "mz:hierarchy_label_country":0,
```
Comment (stepps00) [inline data/115/933/950/3/1159339503.geojson]: Overlay = set label flags to 1 for all placetypes here.

File: data/115/933/950/5/1159339505.geojson [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+    "geom:bbox":"126.607082968,37.787705796,128.364919467,38.675361977",
+    "geom:latitude":38.283838,
+    "geom:longitude":127.464211,
+    "iso:country":"XX",
+    "lbl:latitude":38.324857,
+    "lbl:longitude":127.464589,
+    "mps:latitude":38.324857,
+    "mps:longitude":127.464589,
+    "mz:hierarchy_label":1,
+    "mz:hierarchy_label_country":0,
```
Comment (stepps00) [inline data/115/933/950/5/1159339505.geojson]: Overlay = set label flags to 1 for all placetypes here.

File: data/115/933/948/3/1159339483.geojson [hunk 9]
```diff
@@ -0,0 +1,163 @@
...
+    "geom:bbox":"-70.631612255,-23.8375797185,-67.013966838,-17.506588198",
+    "geom:latitude":-21.43361,
+    "geom:longitude":-69.170543,
+    "iso:country":"XX",
+    "lbl:latitude":-21.912998,
+    "lbl:longitude":-69.142974,
+    "mps:latitude":-21.912998,
+    "mps:longitude":-69.142974,
+    "mz:hierarchy_label":1,
+    "mz:hierarchy_label_country":0,
```
Comment (stepps00) [inline data/115/933/948/3/1159339483.geojson]: Let's not play this one up, set to 1 for all.

File: data/115/933/951/9/1159339519.geojson [hunk 10]
```diff
@@ -0,0 +1,189 @@
...
+    "geom:bbox":"-61.396712809,1.185820211,-58.149281378,8.558010158",
+    "geom:latitude":5.103315,
+    "geom:longitude":-59.485633,
+    "iso:country":"XX",
+    "lbl:latitude":6.094247,
+    "lbl:longitude":-59.644771,
+    "mps:latitude":6.094247,
+    "mps:longitude":-59.644771,
+    "mz:hierarchy_label":1,
+    "mz:hierarchy_label_country":0,
```
Comment (stepps00) [inline data/115/933/951/9/1159339519.geojson]: Set all labels to 1.

File: data/115/933/953/7/1159339537.geojson [hunk 11]
```diff
@@ -0,0 +1,217 @@
...
+    "geom:bbox":"-89.236512208,15.879651999,-87.507679817,17.437639853",
+    "geom:latitude":16.595727,
+    "geom:longitude":-88.690479,
+    "iso:country":"XX",
+    "lbl:latitude":16.5722,
+    "lbl:longitude":-88.598948,
+    "mps:latitude":16.5722,
+    "mps:longitude":-88.598948,
+    "mz:hierarchy_label":1,
+    "mz:hierarchy_label_country":0,
```
Comment (stepps00) [inline data/115/933/953/7/1159339537.geojson]: set all labels to 1

File: data/856/321/95/85632195.geojson [hunk 12]
```diff
@@ -11,12 +11,19 @@
...
-    "lbl:longitude":-54.179172,
-    "mps:latitude":2.55227,
-    "mps:longitude":-54.179172,
+    "lbl:latitude":2.568854,
+    "lbl:longitude":-54.171127,
+    "mps:latitude":2.568854,
+    "mps:longitude":-54.171127,
     "mz:hierarchy_label":1,
-    "mz:is_current":-1,
+    "mz:hierarchy_label_country":0,
```
Comment (stepps00) [inline data/856/321/95/85632195.geojson]: Set labels to 1.

File: data/115/933/951/7/1159339517.geojson [hunk 13]
```diff
@@ -0,0 +1,217 @@
...
+    "geom:bbox":"-5.398858679,35.847357489,-5.283070442,35.926174221",
+    "geom:latitude":35.892449,
+    "geom:longitude":-5.347382,
+    "iso:country":"XX",
+    "lbl:latitude":35.887415,
+    "lbl:longitude":-5.348259,
+    "mps:latitude":35.887415,
+    "mps:longitude":-5.348259,
+    "mz:hierarchy_label":1,
+    "mz:hierarchy_label_country":0,
```
Comment (stepps00) [inline data/115/933/951/7/1159339517.geojson]: This and other labels to 1.

File: data/115/933/952/3/1159339523.geojson [hunk 14]
```diff
@@ -0,0 +1,217 @@
...
+    "geom:bbox":"-2.966920126,35.267874248,-2.912912564,35.329779364",
+    "geom:latitude":35.295351,
+    "geom:longitude":-2.944446,
+    "iso:country":"XX",
+    "lbl:latitude":35.294723,
+    "lbl:longitude":-2.946759,
+    "mps:latitude":35.294723,
+    "mps:longitude":-2.946759,
+    "mz:hierarchy_label":1,
+    "mz:hierarchy_label_country":0,
```
Comment (stepps00) [inline data/115/933/952/3/1159339523.geojson]: This and other labels to 1.

File: data/115/933/948/5/1159339485.geojson [hunk 15]
```diff
@@ -0,0 +1,217 @@
...
+    "geom:latitude":31.390311,
+    "geom:longitude":34.35191,
+    "iso:country":"XX",
+    "lbl:latitude":31.305297,
+    "lbl:longitude":34.28063,
+    "mps:latitude":31.305297,
+    "mps:longitude":34.28063,
+    "mz:hierarchy_label":1,
+    "mz:hierarchy_label_country":0,
+    "mz:hierarchy_label_county":0,
```
Comment (stepps00) [inline data/115/933/948/5/1159339485.geojson]: region and down to 1 (country still 0).

File: data/115/933/948/7/1159339487.geojson [hunk 16]
```diff
@@ -0,0 +1,217 @@
...
+    "geom:latitude":31.945385,
+    "geom:longitude":35.244956,
+    "iso:country":"XX",
+    "lbl:latitude":32.07973,
+    "lbl:longitude":35.25173,
+    "mps:latitude":32.07973,
+    "mps:longitude":35.25173,
+    "mz:hierarchy_label":1,
+    "mz:hierarchy_label_country":0,
+    "mz:hierarchy_label_county":0,
```
Comment (stepps00) [inline data/115/933/948/7/1159339487.geojson]: region and down to 1 (country still 0).

File: data/115/933/955/9/1159339559.geojson [hunk 17]
```diff
@@ -0,0 +1,217 @@
...
+    "geom:bbox":"-61.318186002,-52.935398045,-57.734283007,-51.013197524",
+    "geom:latitude":-51.732872,
+    "geom:longitude":-59.381095,
+    "iso:country":"XX",
+    "lbl:latitude":-51.608913,
+    "lbl:longitude":-58.738602,
+    "mps:latitude":-51.608913,
+    "mps:longitude":-58.738602,
+    "mz:hierarchy_label":1,
+    "mz:hierarchy_label_country":0,
```
Comment (stepps00) [inline data/115/933/955/9/1159339559.geojson]: do show the country and all placetpyes – we rely on the preferred names to carry both claimants in the parenthetical.

File: data/115/933/949/9/1159339499.geojson [hunk 18]
```diff
@@ -0,0 +1,217 @@
...
+    "geom:bbox":"42.647246541,7.996515605,48.9391119999,11.532945054",
+    "geom:latitude":9.731139,
+    "geom:longitude":46.254617,
+    "iso:country":"XX",
+    "lbl:latitude":9.443889,
+    "lbl:longitude":46.731595,
+    "mps:latitude":9.443889,
+    "mps:longitude":46.731595,
+    "mz:hierarchy_label":1,
+    "mz:hierarchy_label_country":0,
```
Comment (stepps00) [inline data/115/933/949/9/1159339499.geojson]: Let's do let thru country and all labels.

**Comment by stepps00** [comment]
@nvkelso - this PR is ready for review.

**Comment by nvkelso** [comment]
Please update the country &/or dependency names as well as it's those properties (not the `disputed` names) that drive label generation. For example:

`name:eng_x_preferred`: **Falkland Islands (Islas Malvinas)**
`name:esp_x_preferred`: **Islas Malvinas (Falkland Islands)**

**Comment by nvkelso** [comment]
Please also... Others joint parenthetical names besides Falkland ([src](https://www.cia.gov/library/publications/the-world-factbook/fields/2070.html)):

- Andean Southern Ice Field (Campo de Hielo Sur), which it looks like we have as a region but need also as a `disputed`

File: data/115/933/952/5/1159339525.geojson [hunk 19]
```diff
@@ -179,7 +179,12 @@
     "src:geom":"naturalearth",
     "src:population":"naturalearth",
     "src:population:date":"1999",
-    "wof:belongsto":[],
+    "wof:belongsto":[
+        102191575,
+        85633793,
+        "136253057",
```
Comment (nvkelso) [inline data/115/933/952/5/1159339525.geojson]: Watch out for quoted ints.

File: data/115/933/952/5/1159339525.geojson [hunk 20]
```diff
@@ -191,9 +196,21 @@
     ],
     "wof:country":"XX",
     "wof:geomhash":"facdfbe05da330cb0a36ae090872c390",
-    "wof:hierarchy":[],
+    "wof:hierarchy":[
+        {
+            "continent_id":102191575,
+            "country_id":85633793,
+            "disputed_id":1159339525,
+            "empire_id":"136253057"
```
Comment (nvkelso) [inline data/115/933/952/5/1159339525.geojson]: Watch quoted ints.

File: data/115/933/953/3/1159339533.geojson [hunk 21]
```diff
@@ -185,7 +185,13 @@
     ],
     "src:population":"naturalearth",
     "src:population:date":"2017",
-    "wof:belongsto":[],
+    "wof:belongsto":[
+        102191581,
+        85633129,
+        "102191581",
```
Comment (nvkelso) [inline data/115/933/953/3/1159339533.geojson]: Watch for quoted ints.

File: data/115/933/953/3/1159339533.geojson [hunk 22]
```diff
@@ -185,7 +185,13 @@
...
+        85633159,
+        "136253055"
```
Comment (nvkelso) [inline data/115/933/953/3/1159339533.geojson]: Watch for quoted ints.

File: data/115/933/953/5/1159339535.geojson [hunk 23]
```diff
@@ -182,7 +182,13 @@
     "src:geom":"naturalearth",
     "src:population":"naturalearth",
     "src:population:date":"2013",
-    "wof:belongsto":[],
+    "wof:belongsto":[
+        "102191581",
+        85632685,
+        "874393555",
```
Comment (nvkelso) [inline data/115/933/953/5/1159339535.geojson]: Watch for quoted ints.

File: data/115/933/953/5/1159339535.geojson [hunk 24]
```diff
@@ -194,9 +200,21 @@
     ],
     "wof:country":"XX",
     "wof:geomhash":"361e3bff9d6c24e441635533cd5e5527",
-    "wof:hierarchy":[],
+    "wof:hierarchy":[
+        {
+            "continent_id":"102191581",
```
Comment (nvkelso) [inline data/115/933/953/5/1159339535.geojson]: Watch for quoted ints.

File: data/115/933/954/7/1159339547.geojson [hunk 25]
```diff
@@ -193,9 +199,21 @@
     ],
     "wof:country":"XX",
     "wof:geomhash":"0da3c9caf36b15f11093a99ec76dcb26",
-    "wof:hierarchy":[],
+    "wof:hierarchy":[
+        {
+            "continent_id":"102191581",
```
Comment (nvkelso) [inline data/115/933/954/7/1159339547.geojson]: Watch for quoted ints.

File: data/115/933/954/7/1159339547.geojson [hunk 26]
```diff
@@ -193,9 +199,21 @@
...
+            "country_id":85632685,
+            "disputed_id":1159339547,
+            "empire_id":"874393555"
```
Comment (nvkelso) [inline data/115/933/954/7/1159339547.geojson]: Watch for quoted ints.

File: data/115/933/954/9/1159339549.geojson [hunk 27]
```diff
@@ -170,7 +170,13 @@
     "src:geom":"naturalearth",
     "src:population":"naturalearth",
     "src:population:date":"2017",
-    "wof:belongsto":[],
+    "wof:belongsto":[
+        102191581,
+        85633147,
+        "136253037",
```
Comment (nvkelso) [inline data/115/933/954/9/1159339549.geojson]: Watch for quoted ints.

File: data/115/933/954/9/1159339549.geojson [hunk 28]
```diff
@@ -182,9 +188,21 @@
     ],
     "wof:country":"XX",
     "wof:geomhash":"5884e5b2f06bcf1003a8d94f4df3f576",
-    "wof:hierarchy":[],
+    "wof:hierarchy":[
+        {
+            "continent_id":102191581,
+            "country_id":85633147,
+            "disputed_id":1159339549,
+            "empire_id":"136253037"
```
Comment (nvkelso) [inline data/115/933/954/9/1159339549.geojson]: Watch for quoted ints.

**Pull request merged**